src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/WorkflowRun.kt

```diff
@@ -29,7 +29,8 @@ data class WorkflowRun(
     val reports: List<WorkflowReportWithParams>,
     val instances: Map<String, String>,
     val gitBranch: String? = null,
-    val gitCommit: String? = null
+    val gitCommit: String? = null,
+    val workflowRunReports: List<WorkflowRunReport>? = null
```

Comment:
I don't think we need this to be a new val, `reports` should just become a list of `WorkflowRunReports` - and there isn't any need for it to be optional. I think this class is only used when adding the run to the repository, and when fetching the workflow run details, and in both these cases we know what the keys are. 

We do still need to retain the `WorkflowReportWithParams` class since it is required for `WorkflowRunRequest`